source/helpers/components/toc.ts

```diff
@@ -13,7 +15,7 @@ export const getDocTocData = (data: TOCItem) => {
   )[0];
   const href = data[text] as string;
   const expanded = Boolean(data[':expanded']);
-  return { text, href, expanded };
+  return { text: markdownInline(text), href, expanded };
```

Comment:
Without this the backticks in the nav names where not being turned into `<code>`

Reply:
@jerivas I ended up reverting this in https://github.com/oddbird/sass-site/pull/42/commits/b47c9c59ef4080455a020c421c36b630ba2f7cd4, because parsing through markdown was turning `--` into `—` and I don't think we want that.

![Screenshot 2023-05-26 at 7 59 20 AM](https://github.com/oddbird/sass-site/assets/552316/3c52b408-31d1-41ae-8bb0-1a68ed20c77e)